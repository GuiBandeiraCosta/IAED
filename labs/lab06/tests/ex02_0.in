a 1
b
c 1:2:3
d 1:2
e Ze Manel:1:2:3
d 7:8
c 4:5:6
f