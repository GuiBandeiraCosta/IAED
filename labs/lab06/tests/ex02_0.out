Bem Vindo passageiro 1
OK!
O Passageiro 1 esta sentado na fila 2, lugar 3
Ha lugares disponiveis na fila 1, lugar 2
Bem vindo Ze Manel. O seu numero identificador eh 1 e esta sentado na fila 2, lugar 3
Ha lugares disponiveis na fila 7, lugar 8
O Passageiro 4 esta sentado na fila 5, lugar 6